cw_monero/lib/monero_wallet_service.dart

```diff
@@ -82,12 +88,14 @@ class MoneroWalletService extends WalletService<
         final polyseed = Polyseed.create();
         final lang = PolyseedLang.getByEnglishName(credentials.language);
 
+        if (credentials.passphrase != null) polyseed.crypt(credentials.passphrase!);
```

Comment:
just to make sure I understand this correctly, so this will encrypt the polyseed, and then in `_restoreFromPolyseed` we would decrypt it so that we create the wallet right?
but then we save the seed after it's being decrypted, so wouldn't that show the unencrypted seed?

Reply:
@konstantinullrich